src/NuGetForUnity/Editor/Ui/NugetWindow.cs

```diff
@@ -980,29 +1040,32 @@ private void DrawPackage(INugetPackage package, GUIStyle packageStyle, GUIStyle
                         }
                     }
 
-                    var versionComparison = installed.PackageVersion.CompareTo(package.PackageVersion);
-                    if (versionComparison < 0)
+                    if (currentTab == NugetWindowTab.UpdatesTab)
```

Comment:
Se above I would allow to update package even on Installed / Online tab. See Visual Studio package window. Or is the UI to bloated?